examples/sign.rs

```diff
@@ -23,12 +23,13 @@ SOFTWARE.
 */
 
 use cloudfront_policy_signer;
+use cloudfront_policy_signer::CloudFrontCannedPolicySigner;
 
 fn main() {
     // Replace me with actual URI
     let resource = "https://example.cloudfront.net/flowerpot.png";
     // Replace me with actual desired expiry time
-    let expiry = 1579532331;
+    let expiry = 2579532331;
```

Comment:
This is now somewhere in 2051, the old value would never work (since that was in 2020), so the download url always returned access denied